src/encoded/types/quality_metric.py

```diff
@@ -62,6 +103,10 @@ class QualityMetricFastqc(QualityMetric):
     schema = load_schema('encoded:schemas/quality_metric_fastqc.json')
     embedded_list = QualityMetric.embedded_list
 
+    @calculated_property(schema=QC_SUMMARY_SCHEMA)
+    def quality_metric_summary(self, request):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5599003)